organizational/conduct.rst

```diff
@@ -0,0 +1,101 @@
+- Sexual images in public spaces
+- Deliberate intimidation, stalking, or following
+- Harassing photography or recording
+- Sustained disruption of talks or other events
+- Inappropriate physical contact
+- Unwelcome sexual attention
+- Advocating for, or encouraging, any of the above behavior
+
+Necessarily, the goals of learning, community, and equality are in tension
+with the goals individual empowerment, freedom of thought, and freedom of
+expression. We strive for politeness and civil discourse but we also strive to
+make all welcome. Even those with unpopular ideas or thoughts. However, in
+order to foster our community certain limits on free expression must be
+observed. We expect that debate and constructive criticism are part of this
+open atmosphere. We expect that some members of the community may express
+ideas which are not politically correct. We expect that some ideas expressed
+by a minority may be deeply unpopular and distasteful to the majority.
+However, if these ideas or discussions devolve into attacks against a person
+and/or their intrinsic characteristics, we believe that this is unacceptable.
+
+Enforcement
+-----------
+
+This policy applies at all Hacker Society events and spaces, online and
+offline.  This includes, but is not limited to:
+
+- Weekly talks
+- Open hacks
+- Link-State
+- HackCWRU
+- Mailing lists (hackers-discuss, cwru-hackers)
+- GitHub organization
+- Slack team
+
+The policy applies to attendees as well as any speakers, sponsors, or other
+participants in these activities and communication streams.  Although we don't
+have any direct authority over behavior in our typical spaces such as the
+Glennan Student Lounge, we expect that HacSoc participants will adhere to these
+guidelines, and we also note that `Case's policies
+<https://students.case.edu/handbook/policy/>`_ are always in force.
+
+We expect participants asked to stop any harassing behavior to comply
+immediately.  If participants do not comply, the Maintainer and HacSoc officers
+retain the right to take any actions to keep HacSoc a welcoming environment for
+all participants.  This includes warning the offender, or removing them from the
+event or communication streams they are abusing.  This also includes further
+measures, such as pursuing University action, should it become necessary.
+
+Reporting
+---------
+
+If someone is making you feel unsafe or unwelcome at HacSoc, please report it as
+soon as possible.  At events such as Link-State and HackCWRU, staff can be
+identified by their shirts.  At other events, you can talk to the officers or
+maintainer.  And you may always report harassing behavior to officers via email
+(hacsoc-officers@case.edu) or via private message on Slack.
+
+Other Policies
+--------------
+
+As mentioned above, the policies of the `CWRU Handbook
+<https://students.case.edu/handbook/policy/>`_ are always applicable at HacSoc
+events and in HacSoc gathering spaces.  Other policies that we believe apply
```

Comment:
I was the one who wrote "we believe they apply".  One reason because my writing style was dancing around a bit, which is my bad :/.  But the other reason I had to be hesitant was that these ACM and IEEE links aren't codes of conduct per se.  They are statements of ethics that are a ton broader than "how do we expect people to behave in HacSoc gatherings."  While most of what these are good things that I agree with, I think the purpose of this document is to lay out expectations for behavior in HacSoc, not lay out a moral or ethical framework for our members.  Also, I haven't taken the time to fully read and digest these policies, and I'm not going to say "these certainly apply" without doing so.